docs/src/components/Layout/Page/Page.component.tsx

```diff
@@ -4,7 +4,8 @@ import * as React from 'react';
 import classNames from 'classnames';
 import { MDXProvider } from '@mdx-js/tag';
 import { AutoComplete } from '@retailmenot/anchor';
-import styled, { css, ThemeProvider } from '@xstyled/styled-components';
+import styled, { ThemeProvider } from '@xstyled/styled-components';
+import { css } from 'styled-components';
```

Comment:
This seems like a step backward.

Reply:
Real problem is that I'm not 100% sure what is causing the issue. I have a gut feeling this may be it, but until I can hit the deploy endpoint where it's failing I can't tell. This is more of a step to figuring out the cause. If this does fix it, then the next step is figuring out how to get `css` from `xstyled` to work.

Reply:
Can't you run the CI scripts locally minus the deploy?

Reply:
https://github.com/RetailMeNot/anchor/blob/master/Dockerfile

`docker build -t client .`
`docker run client`